payments-core/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -69,6 +69,12 @@ sealed interface StripeIntent : StripeModel {
      */
     val linkFundingSources: List<String>
 
+    /**
+     * Payment method options for this StripeIntent.
+     */
+    @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    val paymentMethodOptionsMap: Map<String, PaymentMethodOptions>?
```

Comment:
```suggestion
    val paymentMethodOptions: Map<String, PaymentMethodOptions>? = emptyMap(),
```

Reply:
Since StripeIntent is an interface, we cannot have default values. Also, cannot use the name `paymentMethodOptions` as this would conflict with the PaymentIntent `getPaymentMethodOptions` function